frontend/view-edit-lead/edit-lead-inputs/lead-services-inputs.component.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import easyFetch from "../../util/easy-fetch";
+import { IntakeService } from "../../util/services-inputs.component";
+import IntakeServicesInputs from "../../util/services-inputs.component";
+
+export default function LeadServicesInformationInputs(
+  props: LeadServicesInformationInputsProps
+) {
+  const [services, setServices] = React.useState([]);
+  const [leadServices, setLeadServices] = React.useState(
+    props.lead.leadServices
+  );
+  const leadServicesRef = React.useRef(null);
+
+  React.useEffect(() => {
+    const abortController = new AbortController();
+
+    easyFetch("/api/services", { signal: abortController.signal }).then(data =>
+      setServices(data.services)
+    );
```

Comment:
To ensure that the error makes it to the window error handler and a toast is shown:

```suggestion
    ).catch(err => {
      setTimeout(() => {
        throw err
      })
    })
```